### ISSUE TITLE
setup for qtcreator on windows

### ISSUE THREAD

**Issue by sjoerdtimmer** [DocxFactory/DocxFactory#12]
I'm trying to configure qtcreator to compile and link docxfactory but without luck so far.
Any hints would be appreciated. 

I have started a new test project which does not even reference any of the docxfactory things yet.
I have tried many combinations of QMAKE_LFLAGS, LIBS and QMAKE_LIBS in my qtcreator project (.pro) file. U believe at least my INCLUDEPATH is correctly set at:
```
INCLUDEPATH += "C:\\DocxFactory\\include"
```

When compiling I get many undefined reference errors for functions starting with _impl_ at best or errors about it not being able to find things. (For the record I've had to move my DocxFactory installation to C:\DocxFactory because the space in Program Files is a nightmare to escape since qmake preprocesses quotes in many variables differently)

One problem I could think of is the fact that I'm using a 32 bits mingw compiler as my qtcreator toolkit on a 64 bit windows installation. I wasn't sure if this is ok, and if my guess that this would require a 64 bits docxfactory  is correct. 

At this point I have many unknowns and could really use some hints or tips...

**Comment by sjoerdtimmer** [comment]
right clicking the pro file there appears to be a gui to add libraries. I selected "external library" and it added the folowing lines:
```
win32: LIBS += -L$$PWD/'../../../../Program Files (x86)/DocxFactory/lib/' -lDocxFactory
INCLUDEPATH += $$PWD/'../../../../Program Files (x86)/DocxFactory/include'
DEPENDPATH += $$PWD/'../../../../Program Files (x86)/DocxFactory/include' 
```
Which appears to work as far as compiling and linking is concerned.

However, when I run the application it prints:
```
The program has unexpectedly finished.
The process was ended forcefully.
```
When in debugging mode a popup is shown with the error:
```
During startup program exited with with code 0xc00007b
```
( I think 0xc00007b is probably -1 in 2's complement or something like that)

**Comment by sjoerdtimmer** [comment]
In case anyone is interested:
It is not possible to link against a msvc compiled lib using mingw. In hindsight that is obvious.
I was able to use docxfactory with qtcreator after installing visual studio and installing the msvc components from the qt installation framework. For the record: I used 64 bits visual studio 2017 community edition (free) and Qt 5.9.2 on windows 10. I had some trouble getting msvc to compile and link my existing project and I ended up deleting all build- directories before it worked but for new projects it seems to work out of the box.
